modules/pipeline/providers/definition/pipeline_definition.go

```diff
@@ -273,3 +273,14 @@ func (p pipelineDefinition) StaticsGroupByRemote(ctx context.Context, request *p
 	}
 	return &pb.PipelineDefinitionStaticsResponse{PipelineDefinitionStatistics: pipelineDefinitionStatistics}, nil
 }
+
+func (p pipelineDefinition) ListRef(ctx context.Context, req *pb.PipelineDefinitionRefListRequest) (*pb.PipelineDefinitionRefListResponse, error) {
```

Comment:
I suggest add Ref -> UsedRef is more precise.

Reply:
done